### DISCUSSION TITLE
How to use Volar Language Server?

### DISCUSSION THREAD

**Discussion by arun-mani-j** [emacs-lsp/lsp-mode#4078] [Q&A] [answered]
Hello all!
I'm on Doom Emacs 28.2
I manually installed Volar and `M-x lsp-install-server` says it is already installed (`vue-semantic-server`). However if I open any `.vue` file, it shows this:
![image](https://github.com/emacs-lsp/lsp-mode/assets/49952138/9593663a-c2f0-47d2-97b7-bd6c9bcfdc96)

Where did I mess up? 0_0

(If I type `M-x lsp-version` it says `Wrong type argumenet: package-desc, nil`, that's why couldn't attach the version...)

**Comment by sevillaarvin** [comment, marked as answer]
Hi, try adding `.volarrc` in the root of your project directory.

https://github.com/emacs-lsp/lsp-mode/blob/ee58d9eb66f85c2258b9745f721965f0d62071fb/clients/lsp-volar.el#L99C4-L99C4

**Reply by arun-mani-j** [reply]
Thanks, this works! It doesn't turn on automatically but `m-x lsp` does the job and that's fine for me (I wouldn't want Volar all the time :sweat_smile: )
